server/server-glue.c

```diff
@@ -1,7 +1,7 @@
 #include "server-glue.h"
 
 #include "frida-core.h"
-#ifdef HAVE_IOS
+#if defined (HAVE_IOS) || defined (HAVE_TVOS)
 # include "server-ios.h"
```

Comment:
We need to rename this and `_frida_server_ios_configure()` to avoid making this confusing.